fatiando/gravmag/transform.py

```diff
@@ -118,6 +125,97 @@ def tga(x, y, data, shape, method='fd'):
     return res
 
 
+def mstde(x, y, data, shape):
+    """
+    Multi-Scale Tilt Depth Estimation
+
+    Implements the method of van Buren (2013).
+
+    An empirical method for attempting to approximate the depth of the source
+    of a magnetic anomaly, based on altering tilt and continuation of the
+    magnetic field intensity.
+
+    This method will also work well on Total Gradient Amplitude, but care must
+    be taken to reduce noise.
+
+    .. warning::
+
+        If the data is not in SI units, the derivatives will be in
+        strange units! I strongly recommend converting the data to SI
+        **before** calculating the derivatives (use one of the unit conversion
+        functions of :mod:`fatiando.utils`). This way the derivative will be in
+        SI units and can be easily converted to what unit you want.
+
+    **References**
+
+    Van Buren, Reece. 2013. “Multi-Scale Tilt Depth Estimation.” MSc Thesis,
```

Comment:
These two " characters are causing errors. They are no ASCII characters (probably from copying from a PDF file). You should replace them with the proper " from your keyboard or just remove them.